### PR TITLE
✨ (AI-int): Integrate AI component

### PR DESCRIPTION
**Pull request by LarisaBothaTuks**
Transfer and integrate XGBoost model into repository

### CODE REVIEW

**Comment by LarisaBothaTuks** [comment]
Things needing to be discussed regarding AI:

retraining
global parameters
inviteID generation
how the AI model is going to be used
data prep structure
env file
features
model generation after feature analysis

File: data-prep/app/routes.py [hunk 1]
```diff
@@ -1,9 +1,21 @@
...
     return "hello world"
 
 @app.route("/predict")
 def predict():
     return "SUPER CLEVER AI DATA"
+
+@app.route("/featureAnalysis")
+def featureAnalysis():
+    ms = hello
+    return ms
```
Comment (KyleSmith19091) [inline data-prep/app/routes.py:16]: So these functions need to return a string, so we possible have the data encoded into JSON.

**Comment by LarisaBothaTuks** [comment]
Should I add the predictOne function? I do not see the need for it?

**Comment by KyleSmith19091** [comment]
Yes add the predictOne, might need it in the future

**Review by KyleSmith19091 — APPROVED** [review]

**Pull request merged**